openwisp_notifications/templates/openwisp_notifications/emails/notification.html

```diff
@@ -137,16 +130,21 @@
       <tbody>
         <tr>
           <td>
-            <div>
-              <p class="timestamp">{{ notification.timestamp|date:"F j, Y, g:i A e" }}</p>
-            </div>
             <div>
               <span class="badge {{ notification.level }}">{{ notification.level }}</span>
-              <span class="title"><p>{{ notification.email_message|striptags }}</p></span>
+              <div class="title"><p>{{ notification.email_message|striptags }}</p></div>
             </div>
           </td>
           <td class="right-arrow-container">
-            <img src="{{ site_url }}{% static 'ui/openwisp/images/right-arrow.png' %}" alt="right-arrow">
+            <img class="right-arrow" src="{{ site_url }}{% static 'ui/openwisp/images/right-arrow.png' %}" alt="right-arrow">
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <hr>
```

Comment:
What about:
```suggestion
            <div class="hr"></div>
```

Related to https://github.com/openwisp/openwisp-utils/pull/493#pullrequestreview-3108055490.